data/mods/gen4/abilities.js

```diff
@@ -133,6 +133,27 @@ let BattleAbilities = {
 		inherit: true,
 		rating: 2.5,
 	},
+	"intimidate": {
+		inherit: true,
+		onStart(pokemon) {
+			let activated = false;
+			for (const target of pokemon.side.foe.active) {
+				if (!target || !this.isAdjacent(target, pokemon)) continue;
+				if (!activated) {
+					this.add('-ability', pokemon, 'Intimidate', 'boost');
```

Comment:
I don't know whether the ability should `activate` or not. (ie. does it `activate` and simply not `unboost`, or does it not even `activate` in this scenario? From UPC:

> *that ability will do nothing to the user*

seems like it should activate but do nothing, which is why I've coded it that way.

Reply:
Gen 4 Intimidate actually doesn't activate at all if every target has a substitute, which means the hint is also an information leak.

Reply:
If I understand you correctly, its not just the hint which is an information leak but the code as written today is also leaking information in that case? (ie/ there shouldn't be an `-ability` or `-immune` even).

I've rewritten this so that it should never leak information, but I still think the `-hint` is valid/won't leak provided the `-ability` activates

Reply:
👍